src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -123,13 +112,7 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
       <div className="PosterPreview__categories">{renderedCategories}</div>
 
       <div className="PosterPreview__authorBox">
-        {presenterUser && (
-          <UserProfilePicture
-            containerClassName="PosterPreview__avatar"
-            user={presenterUser}
-            showStatus
-          />
-        )}
+        {authorName}
```

Comment:
This logic was written by an opensource contributor. If we want to use this logic in the future, we would change the implementation to add the presenter when adding the data